exporter/debugexporter/internal/normal/logs.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package normal // import "go.opentelemetry.io/collector/exporter/debugexporter/internal/normal"
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+type normalLogsMarshaler struct{}
+
+// Ensure normalLogsMarshaller implements interface plog.Marshaler
+var _ plog.Marshaler = normalLogsMarshaler{}
+
+// NewNormalLogsMarshaler returns a plog.Marshaler for normal verbosity. It writes one line of text per log record
+func NewNormalLogsMarshaler() plog.Marshaler {
+	return normalLogsMarshaler{}
+}
+
+func (normalLogsMarshaler) MarshalLogs(ld plog.Logs) ([]byte, error) {
+	var buffer bytes.Buffer
+	for i := 0; i < ld.ResourceLogs().Len(); i++ {
+		resourceLog := ld.ResourceLogs().At(i)
+		for j := 0; j < resourceLog.ScopeLogs().Len(); j++ {
+			scopeLog := resourceLog.ScopeLogs().At(j)
+			for k := 0; k < scopeLog.LogRecords().Len(); k++ {
+				logRecord := scopeLog.LogRecords().At(k)
+				logAttributes := writeAttributes(logRecord.Attributes())
+
+				logString := fmt.Sprintf("%s %s", logRecord.Body().AsString(), strings.Join(logAttributes, " "))
```

Comment:
Would it make sense to include the severity text value? (log level) I think this value gives important context about the body message

Reply:
Good point. Can we make an enhancement in a separate PR? There might be other suggestions to improve/change the output and I think it'll be easier to discuss each one separately.

Reply:
Sounds good, let's bring the discussion in another PR/issue